beta-src/src/components/ui/WDDeletePanel.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { Box, Stack } from "@mui/material";
+import { GameState } from "../../state/interfaces/GameState";
+import WDNotification from "./WDNotification";
+
+interface WDDeletePanelProps {
+  notifications: GameState["notifications"];
+}
+
+const WDDeletePanel: React.FC<WDDeletePanelProps> = function ({
+  notifications,
+}): React.ReactElement {
+  return (
+    <Stack direction="column">
+      <Box>
+        <WDNotification notification={notifications[0]} />
```

Comment:
Let's destructure the array `const [ some_name, some_other_name ] = notifications`;

and pass the const down. Just a preference